src/components/Comments.jsx

```diff
@@ -1,6 +1,20 @@
 import dayjs from "dayjs";
+import { deleteCommentByCommentId, getCommentsByArticleId } from "../api/api";
+
+function Comments({ comments, setComments, setErr }) {
+  function handleDeleteButton(commentId, articleId) {
+    return deleteCommentByCommentId(commentId)
+      .then((res) => {
+        return getCommentsByArticleId(articleId);
+      })
+      .then((updatedComments) => setComments(updatedComments))
+      .catch((err) => {
```

Comment:
You need to not refetch these - if other people are deleting or adding comments, it will be really confusing for you to delete something, only to see other disappear or new comments appear